recipe/meta.yaml

```diff
@@ -1,49 +1,69 @@
-{% set version = "2.4.2" %}
+{% set name = "nlopt" %}
+{% set version = "2.7.1" %}
 
 package:
-  name: nlopt
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: nlopt-{{ version }}.tar.gz
-  url: http://ab-initio.mit.edu/nlopt/nlopt-{{ version }}.tar.gz
-  md5: d0b8f139a4acf29b76dbae69ade8ac54
+  url: https://github.com/stevengj/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: db88232fa5cef0ff6e39943fc63ab6074208831dc0031cf1545f6ecd31ae2a1a
+  patches:
+    - pep3149.patch
 
 build:
-  number: 2
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('nlopt', max_pin='x.x') }}
+  missing_dso_whitelist:
+    - "$RPATH/ld64.so.1"  # [s390x]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - autoconf
-    - automake
-    - libtool
-    - cmake         # [win]
+    - cmake
+    - make      # [unix]
     - swig
```

Comment:
Is swig needed now that you disabled all the bindings?

Reply:
I disabled swig